### ISSUE TITLE
Constant loss

### ISSUE THREAD

**Issue by Aayushktyagi** [sthalles/SimCLR-tensorflow#2]
Hey @sthalles ,
Thanks for sharing the code. I was trying to run your code on my data but I am getting constant loss for every epoch. I tried varying temp,  output_dim, epochs but nothing seems to be working out.
Although I am training with a smaller batch size:32 still l was expecting loss will reduce slightly. 

![Screenshot from 2020-05-22 18-35-35](https://user-images.githubusercontent.com/32107805/82670692-3d24cb80-9c5b-11ea-8f9a-c7506a4f48a9.png)

Also, For augmentation colour distortion s = 1 .0 images look like below. 
![Screenshot from 2020-05-22 17-23-31](https://user-images.githubusercontent.com/32107805/82665612-cb944f80-9c51-11ea-8bc7-42af2777b37c.png)
I tried reducing s = 0.5 and lower still no change in the loss.

Have you come across similar kind of problem?  If yes, can you suggest what might be the reason?
Looking forward to hearing from you.

Thanks
 Aayush

**Comment by aynesss** [comment]
@Aayushktyagi  Thanks for question, I hope that they respond you.
Can you tell me how you import your dataset ? I import my dataset with tf.data.Dataset.from_generator() but I find problems to train this code
